Formula/gdal.rb

```diff
@@ -84,7 +83,7 @@ class Gdal < Formula
     depends_on "swig" => :build
   end
 
-  option "without-python", "Build without python2 support"
+  depends_on "libpng"
```

Comment:
This is due to the way current `autocorrect` is implemented. https://github.com/Homebrew/brew/pull/2610

It swaps the two components first encountered which are out of order. Should I try to implement in some other way?